xapian-applications/omega/templates/query

```diff
@@ -37,87 +37,96 @@ $if{$opt{topterms},$include{inc/toptermsjs}}
 input[type=image] {
   border: 0px;
   }
+
+$include{styles.css}
 </style>
 <script type="text/javascript"><!--
 function f2(n) {return (n>9?"":"0")+n;}
 function write_date(time_t) {
 var d=new Date();d.setTime(time_t*1000);
 var D=d.getFullYear()+"-"+f2(d.getMonth()+1)+"-"+f2(d.getDate());
 var T=f2(d.getHours())+":"+f2(d.getMinutes())+":"+f2(d.getSeconds());
-document.write("<span title=\""+D+" "+T+"\">Modified:<br><b>"+D+"</b></span>");
+document.write("<span>Modified:<br/><time datetime=\""+D+" "+T+"\">"+D+"</time></span>");
```

Comment:
If you want a line break inside the `span`, you can do that with something like `span > time { display: block; }`.